src/Enhavo/Bundle/AppBundle/Viewer/TableViewer.php

```diff
@@ -21,7 +25,17 @@ public function getDefaultConfig()
                     'sortable' => false,
                     'move_after_route' => sprintf('%s_%s_move_after', $this->getBundlePrefix(), $this->getResourceName()),
                     'move_to_page_route' => sprintf('%s_%s_move_to_page', $this->getBundlePrefix(), $this->getResourceName())
-                )
+                ),
+                'batch_actions' => array(
```

Comment:
Can we move them also into subspaces

batch:
  actions:
     ...
  route: ...